man/ellipsis-package.Rd

```diff
@@ -6,14 +6,15 @@
 \alias{ellipsis-package}
 \title{ellipsis: Tools for Working with ...}
 \description{
-... is a powerful tool for function extensibility. Unfortunately 
+The ellipsis is a powerful tool for extending functions. Unfortunately 
```

Comment:
These changes must just be from a stale document step? If you want I can remove this file from the PR.